src/main/java/models/Suit.java

```diff
@@ -6,6 +6,6 @@
  * Created by michaelhilton on 1/25/16.
  */
 public enum Suit  implements Serializable {
-    Hearts, Spades, Diamonds, Clubs
+    Hearts, Spades, Diamonds, Clubs, Coins, Cups, Swords
```

Comment:
wouldn't it be better if you made a separate enum for Suit Spanish? I feel like this would deal out all 7 suits.